kafkazk/brokers.go

```diff
@@ -102,6 +103,7 @@ type Broker struct {
 	Replace     bool
 	Missing     bool
 	New         bool
+	RackMissing bool
```

Comment:
Having the added field in `BrokerStatus` makes sense, but will we ever use this in the `Broker` type? Whether this is missing is implied whether the `Locality` field is not empty.